packages/react-instantsearch-core/src/index.js

```diff
@@ -48,3 +48,6 @@ export { default as connectStats } from './connectors/connectStats';
 export {
   default as connectToggleRefinement,
 } from './connectors/connectToggleRefinement';
+export {
+  default as connectHitsInsights,
+} from './connectors/connectHitsInsights';
```

Comment:
```suggestion
} from './connectors/connectHitInsights';
```

We connect only one `hit` to insights.

Reply:
fair point 😄  
done here fd663c96db53ad97231e17f1644e13a7196ab8d5